src/module-emotion.d.ts

```diff
@@ -8,11 +8,10 @@
 // https://github.com/emotion-js/emotion/discussions/2291
 //   https://emotion.sh/docs/typescript#define-a-theme
 
-import { Theme as EmotionTheme } from '@emotion/react';
+// import { Theme as EmotionTheme } from '@emotion/react';
 import { Theme as MaterialUITheme } from '@mui/material';
 
-//import '@emotion/react';
 declare module '@emotion/react' {
     // Re-declare the emotion theme to have the properties of the MaterialUiTheme
-    export interface Theme extends EmotionTheme, MaterialUITheme {}
+    export interface Theme extends /*EmotionTheme,*/ MaterialUITheme {}
```

Comment:
why ?

Reply:
<pre>&gt; gridexplore-app@0.1.0 build
&gt; tsc &amp;&amp; vite build

<font color="#34E2E2">src/module-emotion.d.ts</font>:<font color="#FCE94F">16</font>:<font color="#FCE94F">22</font> - <font color="#EF2929">error</font><font color="#555753"> TS2310: </font>Type &apos;Theme&apos; recursively references itself as a base type.

<span style="background-color:#FFFFFF"><font color="#300A24">16</font></span>     export interface Theme extends EmotionTheme, MaterialUITheme {}
<span style="background-color:#FFFFFF"><font color="#300A24">  </font></span> <font color="#EF2929">                     ~~~~~</font>


Found 1 error in src/module-emotion.d.ts<font color="#555753">:16</font>
</pre>

Reply:
Since we're augmenting emotion's theme here maybe it wasn't relevant to reinject it into itself in the first place